### PR TITLE
March dependabot dependencies update for 16

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -417,6 +416,7 @@
     "identity-obj-proxy": "3.0.0",
     "jest": "26.6.3",
     "jest-environment-jsdom": "^26",
+    "jsonwebtoken": "9.0.0",
```
Comment (davisagli) [inline package.json:419]: @sneridagh A major semver update for jsonwebtoken and moving it to devDependencies are not backwards compatible, if someone was using it downstream without putting it in package.json explicitly. I would leave this one out of the backport.

File: packages/generator-volto/package.json [hunk 2]
```diff
@@ -108,7 +108,6 @@
     "fs-extra": "3.0.0",
     "gitly": "2.0.3",
     "mkdirp-then": "1.2.0",
-    "ms": "1.0.0",
```
Comment (davisagli) [inline packages/generator-volto/package.json:111]: @sneridagh Also technically not backwards compatible, but it's very unlikely that someone is extending the generator so I wouldn't worry about it.

**Review by davisagli — CHANGES_REQUESTED** [review]
@sneridagh Thanks for preparing this. I have a couple doubts about what to include in the backport.

**Comment by sneridagh** [comment]
@davisagli ok, let's wait for this then.

**Comment by davisagli** [comment]
@sneridagh I think it's important to do the updates which are just patch upgrades though, because there are some security fixes.

**Review by davisagli — APPROVED** [review]
@sneridagh I think this is ready now

**Pull request merged**